tern/analyze/common.py

```diff
@@ -222,6 +223,59 @@ def convert_to_pkg_dicts(pkg_dict):
     return pkg_list
 
 
+def get_deb_package_license(deb_copyright):
+    '''
+    Given the debian copyright file location,
+    1. parse the copyright file
+    2. filter out all the available licenses
+    3. returns a list of unique licenses found inside
+    the copyright text
+    '''
+    mounted_path = os.path.join(
+        rootfs.mount_dir, constants.temp_folder, constants.mergedir)
+    deb_copyright = '{0}{1}'.format(mounted_path, deb_copyright)
```

Comment:
This is interesting: what gets returned by `collate_list_metadata` should be a dictionary with a key called `copyrights` and these should be copyright text not copyright file locations. So what `deb_copyright` here is a string that looks like `.tern/temp/mergedir<some text>`. But `DebianCopyright.from_file` requires a file location only.

Reply:
@nishakm if we see this code in the combination of updated `base.yml` with returning copyright file location then it would make complete sense. I did this because as you pointed out that `debut` does not have a way as of now to use the copyright text directly.

Reply:
I would rather not have that change as I would like the enabling on `base.yml` to be as uniform across all package managers as possible.